string name
int32  intValue
float32 floatValue
